content/en/docs/concepts/services-networking/topology-aware-hints.md

```diff
@@ -0,0 +1,154 @@
+---
+reviewers:
+- robscott
+title: Topology Aware Hints
+content_type: concept
+weight: 10
+---
+
+
+<!-- overview -->
+
+{{< feature-state for_k8s_version="v1.21" state="alpha" >}}
+
+_Topology Aware Hints_ enable topology aware routing with topology hints
+included in {{< glossary_tooltip text="EndpointSlices" term_id="endpoint-slice"
```

Comment:
will this linewrap work?  Or do you need to force it after the closing braces?

Reply:
Yep it did work, but I ended up removing this line as part of an earlier edit.